apps/vr-tests-react-components/src/stories/Rating.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { CircleRegular, CircleFilled, SquareRegular, SquareFilled } from '@fluentui/react-icons';
+import { Rating } from '@fluentui/react-rating-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('Rating Converged Color', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('input')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        .hover('input'[4])
+        .snapshot('active', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </StoryWright>
+  ))
+  .addStory('Neutral Rating with half star', () => <Rating defaultValue={3.5} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Brand Rating with half star', () => <Rating color="brand" defaultValue={3.5} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory('Marigold Rating with half star', () => <Rating color="marigold" defaultValue={3.5} />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  });
+
+storiesOf('Rating Converged Size', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addStory('Rating size small', () => <Rating size="small" />, {})
+  .addStory('Rating size medium', () => <Rating size="medium" />, {})
+  .addStory('Rating size large', () => <Rating size="large" />, {})
+  .addStory('Rating size extra-large', () => <Rating size="extra-large" />, {});
+
+storiesOf('Rating Converged Custom Icons', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
```

Comment:
You can just keep them all in the same `storiesOf` since they have the same decorators.

```suggestion
  .addStory('Rating size extra-large', () => <Rating size="extra-large" />, {})
```